<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>601</width>
      <height>411</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="MainLayout" stretch="1,3,1">
     <property name="spacing">
      <number>4</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <layout class="QVBoxLayout" name="LeftLayout">
       <item>
        <widget class="QWidget" name="PlanWidget" native="true">
         <widget class="QWidget" name="verticalLayoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>160</width>
            <height>80</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="PlanLayout"/>
         </widget>
        </widget>
       </item>
       <item>
        <spacer name="Spacer1">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="MiddleLayout" stretch="1,24">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="WeekLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="BlankLabel">
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="MonLabel">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>星期一</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="TueLabel">
           <property name="text">
            <string>星期二</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="WedLabel">
           <property name="text">
            <string>星期三</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="ThuLabel">
           <property name="text">
            <string>星期四</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="FriLabel">
           <property name="text">
            <string>星期五</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="SatLabel">
           <property name="text">
            <string>星期六</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="SunLabel">
           <property name="text">
            <string>星期日</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="TableLayout" stretch="1,7">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="TimeLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="T1Label">
             <property name="text">
              <string>1</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="T2Label">
             <property name="text">
              <string>2</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="T3Label">
             <property name="text">
              <string>3</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="T4Label">
             <property name="text">
              <string>4</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="T5Label">
             <property name="text">
              <string>5</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="T6Label">
             <property name="text">
              <string>6</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="T7Label">
             <property name="text">
              <string>7</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="T8Label">
             <property name="text">
              <string>8</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="T9Label">
             <property name="text">
              <string>9</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="T10Label">
             <property name="text">
              <string>10</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="T11Label">
             <property name="text">
              <string>11</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="T12Label">
             <property name="text">
              <string>12</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="CourseTableLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <layout class="QVBoxLayout" name="C1Layout">
             <property name="spacing">
              <number>0</number>
             </property>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="C2Layout"/>
           </item>
           <item>
            <layout class="QVBoxLayout" name="C3Layout"/>
           </item>
           <item>
            <layout class="QVBoxLayout" name="C4Layout"/>
           </item>
           <item>
            <layout class="QVBoxLayout" name="C5Layout"/>
           </item>
           <item>
            <layout class="QVBoxLayout" name="C6Layout"/>
           </item>
           <item>
            <layout class="QVBoxLayout" name="C7Layout"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="RightLayout">
       <item>
        <widget class="QWidget" name="CourseWidget" native="true">
         <widget class="QWidget" name="verticalLayoutWidget_2">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>30</y>
            <width>160</width>
            <height>80</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="CourseLayout"/>
         </widget>
        </widget>
       </item>
       <item>
        <spacer name="Spacer2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="AddPlan">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>131</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>新建选课计划</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AddCourse">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>10</y>
      <width>135</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>新建课程</string>
    </property>
   </widget>
   <widget class="QWidget" name="ConflictWidget" native="true">
    <property name="geometry">
     <rect>
      <x>219</x>
      <y>470</y>
      <width>161</width>
      <height>91</height>
     </rect>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>161</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="ConflictLayout">
      <item>
       <layout class="QVBoxLayout" name="ConflictLeftLayout">
        <item>
         <widget class="QLabel" name="LeftTitleLabel">
          <property name="text">
           <string>课程</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="Spacer3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="ConflictRightLayout">
        <item>
         <widget class="QLabel" name="RightTitleLabel">
          <property name="text">
           <string>优先度</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="Spacer4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="DeletePlan">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>470</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>删除选课计划</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DeleteCourse">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>470</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>删除课程</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Recommend">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>推荐选课方案</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
